server/server.go

```diff
@@ -1077,6 +1089,30 @@ func requestLogger(log *zap.SugaredLogger, config RequestLoggingConfiguration) g
 	}
 }
 
+func extractLoggingMetadata(ctx context.Context) []any {
```

Comment:
I don't see any reason that the return type needs to be `[]any`. I think it could be `[]string`.

Even better - when I think of metadata like this, the most obvious container is a map. A list seems a bit strange to me unless there's a good reason for it.

Reply:
I changed the return type for this method, and I created another helper method to transform the map into []string, since the metadata is also needed in the `getBaseFields` method with []string type